src/github.ts

```diff
@@ -387,7 +390,7 @@ export class GitHub {
       repo: this.repository.repo,
       num: 25,
       targetBranch,
-      maxFilesChanged: 64,
+      maxFilesChanged: 100, // max is 100
```

Comment:
This bug would have been an issue if more than 64 commits were merged since the last release?

Reply:
Previously, if there were >64 but less than 100 files in the PR, then we could possibly not attribute the PR to a specific path (the GraphQL for PR files is not paginatable). This minor improvement allows us to not have to make the extra backfill API call if there are between 64 and 100 files in the PR.